br/pkg/task/backup.go

```diff
@@ -800,6 +800,10 @@ func DefaultBackupConfig() BackupConfig {
 	if err != nil {
 		log.Panic("infallible operation failed.", zap.Error(err))
 	}
+
+	// disable checksum for backup by default
+	cfg.Checksum = false
```

Comment:
It seems that the function is only called by BRIE via SQL, but it will be replaced by another config.
https://github.com/pingcap/tidb/blob/c550aae6a125e20c2aaa8d4f11a2f090bc8b2b50/pkg/executor/brie.go#L388